worker/peergrouper/worker_test.go

```diff
@@ -731,7 +750,11 @@ func mustNext(c *gc.C, w *voyeur.Watcher) (val interface{}) {
 		c.Logf("mustNext %p", w)
 		ok := w.Next()
 		val = w.Value()
-		c.Logf("mustNext done %p, ok: %v, val: %#v", w, ok, val)
+		if ok {
+			members := val.([]replicaset.Member)
+			val = "\n" + prettyReplicaSetMembers(members)
```

Comment:
Actually val is an interface{}, and output with %v, so this is just a convenience.
Given that this is a test, if the voyer result is not []replicaset.Member, it'll panic.
It is very much intended, and a convenience to output the voyer result if !ok.